features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/metric/RumSessionEndedMetricTest.kt

```diff
@@ -290,7 +290,7 @@ class RumSessionEndedMetricTest {
         // Given
         val sessionEndedMetric =
             SessionEndedMetric(fakeSessionId, fakeStartReason, fakeNtpOffsetAtStart, backgroundEventTracking)
-        val backgroundViewCount = forge.anInt(min = 1, max = events.size)
+        val backgroundViewCount = forge.anInt(min = 1, max = events.size + 1)
```

Comment:
is there any possibility that `events.size ==0` In that case that function will throw a RuntimeException.

Reply:
as @0xnm explained, `@Forgery events: List<ViewEvent>` will not generate empty list, and I made some pressure test with 500 repeats, I didn't see any exception on that.